lizard_auth_server/forms.py

```diff
@@ -42,6 +85,57 @@ def clean(self):
         return new_data
 
 
+class JWTDecryptForm(forms.Form):
+    """Form to decrypt and verify JWT requests."""
+    key = forms.CharField(max_length=1024)
+    message = JWTField(max_length=8192,
+                       allowed_keys=('key', 'domain', 'force_sso_login'))
+
+    def __init__(self, *args, **kwargs):
```

Comment:
Zie ook https://github.com/lizardsystem/lizard-auth-server/pull/45/files voor mijn variant.

Jackie gaat trouwens alles terugdraaien. Geen JWTField maar alles custom in de `validate()`.